pkg/config/config_test.go

```diff
@@ -41,8 +41,7 @@ func TestUnmarshalConfig(t *testing.T) {
 }`,
 			wantSpec: &KubernetesDeploymentSpec{
 				Input: KubernetesDeploymentInput{
-					AutoRollback: true,
```

Comment:
this should be remained 👀

Reply:
No. This test is for checking the UnmarshalJSON function of Config struct.
Our default assigning is done after that and not included in this test.

Reply:
got it 👍 It addressed here [pkg/config/deployment_kubernetes_test.go:76](https://github.com/pipe-cd/pipe/pull/2003/files#diff-0be65797117ef8a7aa8065b9bad17cad1c7aa82469704661753e08b2d2668ac6R76) already. Thanks for your clarification 🙏